styles/styles.css

```diff
@@ -0,0 +1,153 @@
+@import url("./fonts.css");
+
+* {
+  margin: 0 auto;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  box-sizing: border-box;
+  max-width: 1440px;
+  height: 100vh;
+}
+
+.top-section {
+  width: 1440px;
+  height: 900px;
+  justify-content: space-between;
+  padding-top: 47px;
+  background: url(../images/castelmezzano-background-background.png) no-repeat center;
+  background-size: cover;
+  color: #ffffff;
+}
+
+.header {
+  display: flex;
+ }
+
+.logo-img {
+ background-image: url("../images/sprite.png");
+  width: 205px;
+  height: 50px;
+  margin-left: 104px;
+  margin-right: 583px;
+}
+
+.item-1 {
+  font-weight: 400;
+  font-size: 24px;
+  color: #ffffff;
+  margin-right: 24px;
+  text-decoration: none;
+  align-items: center;
+}
+
+.item-2 {
+  background-image: url("../images/sprite.png");
+  width: 30px;
+  background-position-x: -205px;
+  background-repeat: no-repeat;
+  margin-left: 24px;
+}
+
+.item-3 {
+  background-image: url("../images/sprite.png");
+  width: 40px;
+  background-position-x: -235px;
+  background-repeat: no-repeat;
+  margin-left: 24px;
+}
+
+.line {
+  border: 1px solid #f5bd41;
+  width: 50px;
+  margin-left: 998px;
```

Comment:
It works well with static screen resolution only.

Reply:
Is it about responsive web?

Reply:
Yep. It is not required now. But better to avoid such weird margins - it will cause an unexpected shift on smaller screens every time.

Reply:
Ok. But this topic will be in 8th lesson. This is only 6th. I try to follow the instructions in the lessons step by step. This coding is really my first one. I'm ready to learn but not in one day.

Reply:
As I wrote, it is not required now. It is just a remark for your in-depth study.

Reply:
Ok, thanks